v1.9/kubeadm/README.md

```diff
@@ -0,0 +1,99 @@
+# kubeadm
+Official documentation:
+ - https://kubernetes.io/docs/setup/independent/install-kubeadm/
+ - https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/
+
+
+# Reproduce Conformance Results:
+
+## pre-reqs
+- vagrant + virtualbox
+
+## install
+The following Vagrantfile will:
+- setup an Ubuntu 17.10 vm with 4 CPUs and 4GB of RAM
+- install a single node k8s cluster with upstream kubeadm, and Weave 1.6
+- start Sonobuoy /w e2e kube-conformance via [sonobuoy-conformance.yaml](../../sonobuoy-conformance.yaml)
+
+The virtualbox provider is assumed.
+```ruby
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "bento/ubuntu-17.10"
+
+  config.vm.provider "virtualbox" do |vb|
+    vb.cpus   = "4"
+    vb.memory = "4096"
+  end
+
+  config.vm.provision "shell", inline: <<-SHELL
+    #!/bin/sh
+    set -ex
+
+    apt-get update && apt-get install -y apt-transport-https
+    curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+    echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" > /etc/apt/sources.list.d/kubernetes.list
+    apt-get update && apt-get install -y kubelet kubeadm kubectl docker.io
```

Comment:
On my env, it installed docker version `1.6.2` with this command. It's so low a version...

Reply:
@xiangpengzhao did you use the Vagrantfile?
I'm seeing `1.13.1` on my vagrant using `docker version` and `apt show docker.io`.

Reply:
No, I didn't use the Vagrantfile. I followed the steps that of 1.8 tests. I thought it might have the same version issue here. Didn't notice the Vagrantfile :)

Reply:
As long as you have Ubuntu 16.04+ it should be ok I think

Reply:
@luxas I tried to run that apt command on 14.04 and got a lower docker version. Yeah as you said, 16.04 is ok :)

Reply:
This is a good example of why I codified the steps.
There are so many variables and opinions in OS setup.